src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -327,11 +343,132 @@ private ShadowCompareRunsSummaryPane getSummaryPanel() {
         
     }
     
+    /**
+     * Initialized dynamically refresh panel that contains checkbox, textfield and label for auto refreshing.
+     * @return
+     */
+    private JPanel getdynamicallyRefreshPanel() {
+        
+        if (dynamicallyRefreshPanel == null) {
+            dynamicallyRefreshPanel = new JPanel();
+            dynamicallyRefreshPanel.setMaximumSize(new Dimension(700,20));
+            
+            JCheckBox checkbox = new JCheckBox("Refresh Every:");
+            dynamicallyRefreshPanel.add(checkbox);
+            
+            
+            JTextField textField = new JTextField("5",2);
```

Comment:
Perhaps make a string constant variable for the default value ("5" in this case), and place it near the top of this module.  This way it's easy to change